src/test/java/mil/dds/anet/test/resources/LocationResourceTest.java

```diff
@@ -143,4 +143,13 @@ private void updateLocation(Person user) {
     }
   }
 
+  @Test
+  void shouldBeSearchableViaCustomFields() {
+    final LocationSearchQueryInput query =
+        LocationSearchQueryInput.builder().withText("consectetur").build();
+    final AnetBeanList_Location searchObjects =
+        withCredentials(adminUser, t -> queryExecutor.locationList(getListFields(FIELDS), query));
+    assertThat(searchObjects).isNotNull();
+    assertThat(searchObjects.getTotalCount()).isGreaterThan(0);
```

Comment:
You should probably also assert that the found objects match the search text in their customFields (here and in the other server-side tests).